src/Carousel.vue

```diff
@@ -1,48 +1,40 @@
 <template>
-  <div
-    class="VueCarousel"
-    v-bind:class="{ 'VueCarousel--reverse': paginationPosition === 'top' }"
-  >
+  <transition name="fade">
     <div
-      class="VueCarousel-wrapper"
-      ref="VueCarousel-wrapper"
+      :class="rootClass"
+      ref="VueCarousel"
     >
-      <div
-        ref="VueCarousel-inner"
-        :class="[
-          'VueCarousel-inner',
-          { 'VueCarousel-inner--center': isCenterModeEnabled }
-        ]"
-        :style="{
-          'transform': `translate(${currentOffset}px, 0)`,
-          'transition': dragging ? 'none' : transitionStyle,
-          'ms-flex-preferred-size': `${slideWidth}px`,
-          'webkit-flex-basis': `${slideWidth}px`,
-          'flex-basis': `${slideWidth}px`,
-          'visibility': slideWidth ? 'visible' : 'hidden',
-          'height': `${currentHeight}`,
-          'padding-left': `${padding}px`,
-          'padding-right': `${padding}px`
-        }"
-      >
-        <slot></slot>
+      <div class="VueCarousel-wrapper" ref="VueCarousel-wrapper">
+        <div
+          :class="innerClass"
+          :style="innerStyles"
+          ref="VueCarousel-inner"
+        >
+          <slot name="slides" />
```

Comment:
If the slot was provided without a name it sends me back with errors in the console with mixing client and server content (if I remember it well b/c there were a lot of errors)